Dockerfile

```diff
@@ -30,3 +30,4 @@ FROM gcr.io/distroless/static:nonroot
 COPY --from=build --chown=nonroot /go/src/cloudsql-proxy/cloud_sql_proxy /cloud_sql_proxy
 # set the uid as an integer for compatibility with runAsNonRoot in Kubernetes
 USER 65532
+ENTRYPOINT ["/cloud_sql_proxy"]
```

Comment:
Double check: this can be overridden in all of the usual environments (docker, podman, k8s, etc)?

Reply:
- Docker: yes `docker run --entrypoint` 
- Podman: yes, works like docker
- K8s: yes `"container": {"command": "/bin/sh"}`

Reply:
Thanks @hessjcg.